fuzzers/run_fuzzer.py

```diff
@@ -447,6 +448,17 @@ def main(argv):
         os.makedirs(fuzzer_logdir)
     assert os.path.exists(fuzzer_logdir)
 
+    # Set part information from the mapping file and set into the env
+    mapping_file = "../prjxray/mapping.yaml"
```

Comment:
Given that 'XRAY_PART' is defined, you can use 'XRAY_DATABASE'/'XRAY_DATABASE_DIR'/'XRAY_DIR'.

Also I believe this file should be moved to `os.path.join(os.environ['XRAY_DATABASE_DIR'], os.environ['XRAY_DATABASE'])`, so there is that.